internal/policygen/load.go

```diff
@@ -38,6 +39,23 @@ type config struct {
 	Schema    map[string]interface{} `json:"schema,omitempty"`
 }
 
+func ValidateOrgPoliciesConfig(conf map[string]interface{}, allowAdditionalProperties bool) error {
+	if !allowAdditionalProperties {
+		orgPoliciesSchema = append(orgPoliciesSchema, []byte("additionalProperties = false")...)
```

Comment:
It seems to me that this will modify the global variable and will always have `additionalProperties = false` for every subsequent function call. Maybe this should be `:=`, to create a locally scoped variable?

Reply:
Done